package.json

```diff
@@ -16,19 +22,20 @@
     "underscore": "1.7.0"
   },
   "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-cli": "^1.2.0",
-    "grunt-fh-build": "^0.5.0",
-    "grunt-mocha-test": "^0.13.2",
+    "grunt": "~0.4.5",
+    "grunt-cli": "~1.2.0",
+    "grunt-fh-build": "~0.5.0",
+    "grunt-mocha-test": "~0.13.2",
     "istanbul": "0.2.14",
-    "jshint": "^2.5.2",
+    "jshint": "~2.5.2",
     "mocha": "2.4.5",
     "proxyquire": "1.4.0",
-    "sinon": "^1.17.5",
-    "typedoc": "^0.7.1",
+    "sinon": "~1.17.5",
+    "typedoc": "~0.7.1",
+    "typescript": "~2.6.1",
     "valid-url": "1.0.9"
   },
-  "types": "./types/fh-sync.d.ts",
+  "types": "fh-sync.d.ts",
```

Comment:
I moved this due to trouble when making my example. I can move back and try again if we'd prefer to keep it in a subfolder.